web/images/icons-sprite.svg

```diff
@@ -0,0 +1 @@
h-18.376L215.94 31.04l-5.717 17.967 15.11-11.025z" fill-rule="nonzero"/></symbol><symbol viewBox="0 0 19 16" id="twitter"><path fill-rule="evenodd" d="M18.476.3c-.754.464-1.59.8-2.477.98A3.834 3.834 0 0 0 13.154.007c-2.153 0-3.9 1.808-3.9 4.037 0 .316.036.625.102.92-3.24-.17-6.11-1.777-8.033-4.22a4.124 4.124 0 0 0-.528 2.03c0 1.402.69 2.638 1.734 3.36a3.756 3.756 0 0 1-1.765-.506v.05c0 1.956 1.343 3.588 3.126 3.96-.326.09-.67.142-1.026.142-.252 0-.495-.028-.734-.075.496 1.604 1.935 2.772 3.64 2.805A7.648 7.648 0 0 1 0 14.182a10.76 10.76 0 0 0 5.975 1.814c7.17 0 11.09-6.152 11.09-11.488l-.013-.523A8.007 8.007 0 0 0 19 1.898c-.7.32-1.45.54-2.238.636A4.037 4.037 0 0 0 18.476.301z"/></symbol><symbol viewBox="0 0 563.43 563.43" id="user"><path d="M280.79 314.56c83.266 0 150.803-67.54 150.803-150.804S364.055 13.416 280.79 13.416s-150.803 67.537-150.803 150.34S197.524 314.56 280.79 314.56zm0-261.825c61.06 0 111.02 49.96 111.02 111.02s-49.96 111.02-111.02 111.02-111.02-49.958-111.02-111.02 49.958-111.02 111.02-111.02zm-260.9 497.28h523.65c11.1 0 19.89-8.79 19.89-19.89 0-104.083-84.653-189.2-189.198-189.2H189.198C85.116 340.926 0 425.58 0 530.126c0 11.1 8.79 19.89 19.89 19.89zM189.2 380.708h185.034c75.864 0 138.313 56.436 148.028 129.524H41.17c9.714-72.625 72.164-129.524 148.028-129.524z"/></symbol><symbol viewBox="0 0 511.626 511.626" id="wishlist"><path d="M475.366 71.95C451.19 48.342 417.79 36.544 375.15 36.544c-11.8 0-23.842 2.046-36.116 6.136-12.28 4.094-23.702 9.616-34.256 16.563-10.568 6.945-19.65 13.467-27.27 19.556a263.62 263.62 0 0 0-21.695 19.413 264.184 264.184 0 0 0-21.698-19.414c-7.616-6.09-16.702-12.608-27.268-19.557-10.564-6.95-21.985-12.468-34.26-16.562-12.276-4.088-24.317-6.135-36.117-6.135-42.637 0-76.04 11.8-100.21 35.404C12.086 95.55 0 128.285 0 170.16c0 12.753 2.24 25.89 6.71 39.398 4.472 13.514 9.567 25.03 15.276 34.546 5.708 9.514 12.18 18.792 19.414 27.834 7.233 9.04 12.52 15.272 15.846 18.698 3.33 3.426 5.948 5.903 7.85 7.427L243.25 469.938c3.427 3.426 7.614 5.144 12.562 5.144s9.138-1.718 12.563-5.144l177.87-171.31c43.588-43.58 65.38-86.406 65.38-128.472 0-41.877-12.085-74.61-36.26-98.207zm-53.96 199.845L255.812 431.39 89.938 271.508c-35.594-35.585-53.388-69.374-53.388-101.35 0-15.416 2.046-29.027 6.136-40.825 4.093-11.8 9.327-21.177 15.703-28.124 6.376-6.95 14.13-12.607 23.267-16.988 9.14-4.377 18.086-7.328 26.84-8.85 8.754-1.52 18.08-2.28 27.978-2.28 9.896 0 20.557 2.423 31.977 7.278 11.418 4.853 21.934 10.944 31.545 18.27 9.613 7.333 17.845 14.184 24.7 20.558 6.85 6.38 12.56 12.23 17.128 17.56 3.424 4.188 8.09 6.282 13.99 6.282s10.56-2.094 13.99-6.283c4.567-5.33 10.28-11.182 17.13-17.56 6.852-6.373 15.085-13.22 24.694-20.556 9.612-7.328 20.128-13.42 31.552-18.272 11.416-4.854 22.08-7.28 31.977-7.28s19.218.762 27.976 2.282c8.757 1.52 17.702 4.473 26.84 8.85 9.137 4.38 16.892 10.042 23.267 16.988 6.376 6.947 11.612 16.324 15.705 28.124 4.086 11.798 6.132 25.41 6.132 40.824-.002 31.977-17.89 65.86-53.675 101.64z"/></symbol></svg>
```

Comment:
why icon sprite shouldn't be added? It has to be copied from components repository so we need this file inside the theme...